app/views/idv/inherited_proofing/_verify.html.erb

```diff
@@ -0,0 +1,71 @@
+<%#
+locals:
+  pii - user's information
+  remote_identity_proofing - true if we're doing RIDP, false if we're doing IPP
+  step_url - generator for step URLs
+%>
```

Comment:
Can this be using the existing shared template, rather than duplicating it?

https://github.com/18F/identity-idp/blob/main/app/views/idv/shared/_verify.html.erb

Reply:
There were enough differences between the two templates that I felt it would be better to use our own. (For at least the time being.) I would, ideally, like to move us to the existing shared one in the future but for MVP functionality this was the cleanest option.

Reply:
What are the differences? It looks like it may only be the titles, right? We could have a `inheritted_proofing` local or something like that to switch those. I think an even easier approach would be to check with @benjaminchait and @artfulaction if they are okay with using the titles from the existing "Verify your information" screen to keep things consistent.

Reply:
We're not allowing users to edit their addresses or ssn at this time, for example. I could put some conditionals in but it started to look really messy and I didn't want to 💩 up the shared template when I wasn't 100% sure we weren't going to restore editing later.

Reply:
Another minor thing is the the user info should be displayed in all caps.